raiden/tests/integration/api/test_pythonapi_regression.py

```diff
@@ -40,14 +40,21 @@ def run_test_close_regression(raiden_network, deposit, token_addresses):
     # Initialize app2 balance proof and close the channel
     amount = 10
     identifier = 42
+    secret = factories.make_secret()
     assert api1.transfer(
```

Comment:
Humm, this is waiting for ever, the `Timeout` bellow is not every effective, could rename this to `transfer_and_wait` and add the timeout argument?

Reply:
Change `RaidenAPI.transfer` for use in a test? Wouldn't we better put it inside a `with Timeout` block then?